ui/app/styles/core/tabs.scss

```diff
@@ -41,6 +41,10 @@
 
     + * {
       margin-top: 5em;
+
+      &.closer {
+        margin-top: calc(3.5em + 1px);
+      }
```

Comment:
Without some spacing adjustment, it looks like this; a little too far away from the tabs, to me:

<img width="840" alt="image" src="https://user-images.githubusercontent.com/43280/60358666-5914e400-999c-11e9-9203-5f702fa32750.png">

At `3.5em`, the horizontal and vertical spacing are identical, but because of weirdnesses of sight, it looked slightly too close vertically, so I added the extra pixel.

Reply:
I'll allow it 😛